### ISSUE TITLE
no-conditional-in-test conflicts with playwright's official docs

### ISSUE THREAD

**Issue by xWiiLLz** [playwright-community/eslint-plugin-playwright#76]
Hi,

There is a basic use case [described in playwright's docs](https://playwright.dev/docs/api/class-page#page-wait-for-request) on how to wait for specific http requests:
```typescript
// Alternative way with a predicate.
const [request] = await Promise.all([
  // Waits for the next request matching some conditions
  page.waitForRequest(request => request.url() === 'https://example.com' && request.method() === 'GET'),
  // Triggers the request
  page.click('button.triggers-request'),
]);
```

However, the `no-conditional-in-test` appears to be conflicting with this basic use case:
![image](https://user-images.githubusercontent.com/1496661/183957442-c21113a6-b4d4-4091-b215-8fc202debcf6.png)

Is this the expected behaviour for this rule?

Thanks

**Comment by mskelton** [comment]
I think we should reduce the rule's scope to only top level conditionals, rather than **any** conditional.
